models/item.cs

```diff
@@ -0,0 +1,7 @@
+using System;
```

Comment:
It is not clear if this is DB Models or Business Models.

Looking at the usage I believe you mean this to be DB Models.

So rename the folder to be DB Models.